README.md

```diff
@@ -171,10 +171,11 @@ For further details, consult the [wiki](https://github.com/ReactiveX/RxJava/wiki
 
 ## Communication
 
+- [Gitter.im](https://gitter.im/ReactiveX/RxJava)
```

Comment:
Why regrouping? Actually I prefer StackOverflow because questions can be isolated much better. The gitter channels usually invoke chaos by talking over each other and hard to follow who reacted to what.